packages/esm-service-queues-app/src/metrics/metrics-cards/waiting-patients.extension.tsx

```diff
@@ -52,7 +55,9 @@ export default function WaitingPatientsExtension() {
       locationUuid={currentQueueLocation}
       service={currentService?.serviceDisplay}
       serviceUuid={currentService?.serviceUuid}
-      value={initialSelectedItem ? totalCount ?? '--' : serviceCount}>
+      value={initialSelectedItem ? (totalCount ?? '--') : serviceCount}
+      showUrgent={true}
```

Comment:
```suggestion
      showUrgent={urgentCount > 0}
```

Shouldn't we show the `Urgent` label and value pair only if there are actually urgent cases to display?